src/Directory.Build.targets

```
@@ -0,0 +1,7 @@
+<Project>
+
+  <PropertyGroup>
+    <NuGetAuditMode Condition="'$(PackAsTool)' == 'True'">all</NuGetAuditMode>
```

Conditions are case-insensitive, but lowercase is more of the standard these days:

```suggestion
    <NuGetAuditMode Condition="'$(PackAsTool)' == 'true'">all</NuGetAuditMode>
```

---

Setting the property this way does mean that you can't override it for tool projects, in case that's a concern.

---

For anyone int he future - that's true. We can't imagine why we'd want to allow that currently, but a temporary option (before a new reposync could be done) would be to add the Directory.Build.targets file as a reposync exclusion.